plugins/modules/gitlab_group.py

```diff
@@ -202,23 +273,51 @@ def get_group_id(self, group):
     def create_or_update_group(self, name, parent, options):
         changed = False
 
+        payload = {
+            'name': name,
+            'path': options['path'],
+            'visibility': options['visibility'],
+        }
+        if options.get('auto_devops_enabled'):
```

Comment:
This might actually be filtered elsewhere if value is None.

In such a case, most should be moved to payload (line 276).